src/ripple/protocol/Feature.h

```diff
@@ -46,6 +46,23 @@ namespace ripple {
 
 namespace detail {
 
+// *NOTE*
+//
+// Features, or Amendments as they are called elsewhere, are enabled on the
+// network at some specific time based on Validator voting.  Once an
+// Amendment has been enabled for, say, more than two years then retaining
+// the conditional code based on the Amendment is pointless.
```

Comment:
"pointless" is a little strong. They are still useful when replaying old transaction. I agree we should remove them, but I do actually replay old transactions now and then.